src/main/java/appeng/container/implementations/ContainerCraftingCPU.java

```diff
@@ -54,11 +54,11 @@
 public class ContainerCraftingCPU extends AEBaseContainer implements IMEMonitorHandlerReceiver<IAEItemStack>, ICustomNameObject
 {
 
-	final IItemList<IAEItemStack> list = AEApi.instance().storage().createItemList();
-	protected IGrid network;
-	CraftingCPUCluster monitor = null;
-	String cpuName = null;
-	int delay = 40;
+	private final IItemList<IAEItemStack> list = AEApi.instance().storage().createItemList();
+	private IGrid network;
+	private CraftingCPUCluster monitor = null;
+	private String cpuName = null;
+	private final int delay = 40;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "delay" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)